ghproxy/ghmetrics/ghpaths_test.go

```diff
@@ -0,0 +1,112 @@
+package ghmetrics
+
+import "testing"
+
+func Test_getSimplifiedPath(t *testing.T) {
+	type args struct {
+		path string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{"access root path should not fail, is not explicitly handled", args{"/"}, "/"},
+		{"repo branches protection (restrictions for users) by name ", args{"/repos/testOwner/testRepo/branches/testBranch/protection/restrictions/users"}, "/repos/:owner/:repo/branches/:var/protection/restrictions/users"},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := getSimplifiedPath(tt.args.path); got != tt.want {
+				t.Errorf("getSimplifiedPath() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_handleRepos(t *testing.T) {
+	type args struct {
+		path string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{"access root path should not fail, is not explicitly handled", args{"/"}, "/repos"},
```

Comment:
`go vet` should catch this but use named fields in literals please

Reply:
Change coming in the next commit.